frontend/.env.example

```
@@ -1,5 +1,5 @@
-VITE_OPEN_SECRET_API_URL=http://0.0.0.0:3000
+VITE_OPEN_SECRET_API_URL=http://127.0.0.1:3000
 # Public OpenSecret project id. Optional; the app uses this value by default.
 VITE_CLIENT_ID=ba5a14b5-d915-47b1-b7b1-afda52bc5fc6
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The VITE_CLIENT_ID key should go before the VITE_OPEN_SECRET_API_URL key

(UnorderedKey)